### ISSUE TITLE
Impossible selecting item other than first-selected

### ISSUE THREAD

**Issue by arjarn** [lekoala/bootstrap5-tags#94]
I have long item list so I'm using JS for tags parameters.
I wanna multiple selection and possibility to add other values than first-selected.
If i'm using example below, adding new value like 'American' is impossible :-/
Certainly beginner's mistake...

```code
<html lang="fr">
<head>
  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5/dist/css/bootstrap.min.css" rel="stylesheet" />
  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5/dist/js/bootstrap.bundle.min.js" type="module"></script>
    <script type="module">
		import Tags from "https://cdn.jsdelivr.net/gh/lekoala/bootstrap5-tags@master/tags.js";
		
		let items = [
			{ "value": 1, "label": "American" },
			{ "value": 2, "label": "Russian" },
			{ "value": 3, "label": "French" },
			{ "value": 4, "label": "English" }
		];
		let selected = [3,4];
		
		Tags.init("select",
		{
			items: items,
			selected: selected
		});
  </script>
</head>
<body>
<select class="form-select" id="validationTags" name="tags[]" multiple>
   <option selected disabled hidden value="">Choose a tag...</option>
</select>
</body>

**Comment by lekoala** [comment]
@arjarn good catch, passing items is a recent feature and I've missed that part. you can try again with the changes I've made on master it should be much better

**Comment by arjarn** [comment]
Really nice : this is perfect, thanx for your work.

I'm wondering if it's possible having new feature : beforeAdd and beforeClear events with true/false return.
If true add/clear are effective else it cancel operation.
It can be great for having control or confirmation message.

**Comment by lekoala** [comment]
there is a onCanAdd hook that can return false to prevent adding that element. i only check for "false" because it's really meant to prevent something, allowing being the default

in the demo you can check emailCanAdd method and how i show feedback using toasts

i'm not sure about the beforeClear ? either you can clear the value or you can't. Maybe it would be nice to have "set" values than can't be removed

**Comment by lekoala** [comment]
released in https://github.com/lekoala/bootstrap5-tags/releases/tag/1.6.5
